tests/integration/test_saml.py

```diff
@@ -38,8 +38,8 @@ async def test_saml_login(  # pylint: disable=too-many-locals,too-many-arguments
     password = "test-discourse-k8s-password"  # nosecue
     saml_helper.register_user(username=username, email=email, password=password)
 
-    action_result = await run_action(app.name, "create-user", email=email)
-    assert "user" in action_result
+    task = juju.run(app.name + "/0", "create-user", {"email": email})
```

Comment:
```suggestion
    task = juju.run(app.name + "/0", "create-user", params={"email": email})
```

Here and in the other test file.
I could kindof guess the intent behind the third argument, but implicit is better than explicit, isn't it?

Reply:
Re the unit vs app IP: sometimes the app IP address is different, so I'm sticking with unit address like the original tests here.

Re "/0", yes, unit IDs are guaranteed by Juju to be incrementing integers and start at "0", so this is common.

Re `params=`, in Jubilant I haven't made this a keyword arg, 1) because I think it's obvious for `run`, and 2) because it mirrors the CLI, which has params as `key=val` optional positional arguments rather than `--param key=val` or similar.